advanced_java/enum/Day.java

```diff
@@ -0,0 +1,4 @@
+public enum Day {
```

Comment:
Use other Enum in built methods as well and show their functionality here.

Reply:
Could you please name a few?